src/backend/oracle-data-interface/Dockerfile

```diff
@@ -0,0 +1,31 @@
+#############################################################################################
+###              Stage where Docker is building spring boot app using maven               ###
+#############################################################################################
+FROM maven:3.8.3-jdk-11 as build
+
+ARG SKIP_TESTS=true
+ARG ENABLE_SPLUNK=false
+ARG SERVICE_NAME=oracle-data-interface-application
+
+WORKDIR /
+
+COPY . .
+
+RUN mvn -B clean package \
+		-Dtarget.fileName=${SERVICE_NAME} \
+        -Dmaven.test.skip=${SKIP_TESTS} \
+        -Denable-splunk=${ENABLE_SPLUNK}
+
+#############################################################################################
+
+#############################################################################################
+### Stage where Docker is running a java process to run a service built in previous stage ###
+#############################################################################################
+FROM eclipse-temurin:11
+
```

Comment:
Should we be using the [EXPOSE](https://docs.docker.com/engine/reference/builder/#expose) instruction? Typically we have the services always listen internally on 8080.

Reply:
I haven't seen any other Java projects specified EXPOSE instruction. Maybe we can specify it in the main docker-compose.yml but we may ask @KDNeufeld as well if it is a more common way to use EXPOSE in Dockerfile.

Reply:
It's my understanding that EXPOSE is only to help with [ inter-container communication](https://we-are.bookmyshow.com/understanding-expose-in-dockerfile-266938b6a33d) (ie allow one container to reference a service in a different container running on the same docker-compose network) - it doesn't provide any port mapping at all. 

This webapp does run on 8080 - this is default port for all spring-boot projects.  It can be overwritten by specifying a server.port property in application.yml. ie. [example](https://github.com/bcgov/jag-efax/blob/main/src/main/resources/application.yaml#L2)